lib/tomo/plugin/nodenv/tasks.rb

```diff
@@ -0,0 +1,63 @@
+require "shellwords"
+
+module Tomo::Plugin::Nodenv
+  class Tasks < Tomo::TaskLibrary
+    def install
+      run_installer
+      modify_bashrc
+      install_node
+      install_yarn
+    end
+
+    private
+
+    def run_installer
+      install_url = "https://github.com/nodenv/nodenv-installer/raw/master/bin/nodenv-installer"
+      remote.env PATH: raw("$HOME/.nodenv/bin:$HOME/.nodenv/shims:$PATH") do
+        remote.run("curl -fsSL #{install_url.shellescape} | bash")
+      end
+    end
+
+    def modify_bashrc
+      existing_rc = remote.capture("cat", paths.bashrc, raise_on_error: false)
+      return if existing_rc.include?("nodenv init")
+
+      remote.write(text: <<~BASHRC + existing_rc, to: paths.bashrc)
+        if [ -d $HOME/.nodenv ]; then
+          export PATH="$HOME/.nodenv/bin:$PATH"
+          eval "$(nodenv init -)"
+        fi
+      BASHRC
+    end
+
+    def install_node
+      require_setting :nodenv_node_version
+      node_version = settings[:nodenv_node_version]
+
+      unless node_installed?(node_version)
+        logger.info(
+          "Installing node #{node_version} -- this may take several minutes"
+        )
+        remote.run "CFLAGS=-O3 nodenv install #{node_version.shellescape}"
```

Comment:
`CFLAGS=-O3` is useful for ruby-build but I don't think it has an effect for nodenv.

```suggestion
        remote.run "nodenv install #{node_version.shellescape}"
```